mkdocs.yml

```diff
@@ -140,6 +141,7 @@ plugins:
       type: iso_date
       fallback_to_build_date: true
   - social:
+      enabled: false
```

Comment:
If I'm not mistaken, this shouldn't be needed anymore after merging develop into this branch. Right, @mdbenito ?

Reply:
Correct